### PR TITLE
feat(chunkserver): Use aligned memory for writes

### PR DESCRIPTION
**Pull request by lgsilva3087**
The memory for write operations is now aligned to 4 KiB, which allows to later implement disks writes in DirectIO. It also benefits SMR writes by removing the need of memcpy to align the buffer with the underlying drive block size.

The previous approach was also resizing constantly the vector PacketStruct::packet, which implies frequent allocations. The new PacketStruct::alignedBuffer is now reused most of the time without constant allocations.

### CODE REVIEW

**Review by rolysr — APPROVED** [review]
Good change @lgsilva3087. LGTM

**Review by dmga44 — DISMISSED** [review]
LGTM.

File: src/chunkserver/chunkserver_entry.h [hunk 1]
```diff
@@ -43,7 +56,16 @@
 struct PacketStruct {
 	uint8_t *startPtr = nullptr;
 	uint32_t bytesLeft = 0;
+
```
[the remark this answers is not in the document]
Comment (lgsilva3087) [reply src/chunkserver/chunkserver_entry.h]: Removed.

**Review by dmga44 — APPROVED** [review]

**Review by rolysr — APPROVED** [review]

**Review by ralcolea — COMMENTED** [review]
LGTM. Great job @lgsilva3087 :+1: :fire: :muscle:

**Pull request merged**